src/registry/registry.ts

```diff
@@ -31,6 +32,7 @@ const REGISTRY: Registry = {
   checkbox: Checkbox,
   selectboxes: Selectboxes,
   radio: RadioField,
+  textarea: Textarea,
```

Comment:
can you put this in the right order according to the UI?

<img width="397" height="480" alt="Image" src="https://github.com/user-attachments/assets/c83fd53c-bdb8-45cd-8cea-e7b1d1e7ec5c" />

it goes before the checkbox. See also the same structure in the builder repo: https://github.com/open-formulieren/formio-builder/blob/6f3d309259bde6e686b20ea078a8220d22ff16bf/src/registry/index.ts#L68